External/TestDataSet/dataPar.json.json

```diff
@@ -1,6 +1,7 @@
 {
 	"x": [{
 			"name": "ExampleDataSet",
+			"D": {"ROOT": ".\\derivatives\\ExploreASL"},
```

Comment:
the file has two extensions now :)

Also, did you test if these Windows file separators work also in Unix systems?

I wonder if it is not confusing that we use(d) `dataPar.json` in /derivatives/ExploreASL all the time, and now put it somewhere else? Also, this file could now be given a different name, as it contains settings for running ExploreASL rather than data-specific, or we can wait with this?

Reply:
You're right, sorry. I got a bit confused with the different parameter files. I'll move it back to derivatives for now. That's the same way we automatically create them right now during the import workflow. I'd keep the name though, since most of the people are used to it. We can discuss renaming after v1.6.0, okay?

Reply:
Done in 28f180b .